src/luarocks/cmd/nix.lua

```diff
@@ -57,12 +57,15 @@ local function convert2nixLicense(license)
 end
 
 
-local function get_basic_checksum(url)
+local function get_basic_checksum(url, unpack)
```

Comment:
maybe it could be  `get_basic_checksum(command)` instead of `get_basic_checksum(url, unpack)`, what do you think ?

Reply:
sure